packages/loopring_v3/contracts/core/impl/libtransactions/WithdrawTransaction.sol

```diff
@@ -168,6 +169,10 @@ library WithdrawTransaction
             withdrawal.minGas = 0;
 
             delete S.withdrawalRecipient[withdrawal.owner][withdrawal.to][withdrawal.tokenID][withdrawal.amount][withdrawal.nonce];
+        } else if (auxData.isFastWithdrawal > 0) {
+            // process a fastwithdrawal as a normal withdrawal,
+            // write the recipient to prevent the fastwithdrawal being processed by the LP again.
+            S.withdrawalRecipient[withdrawal.owner][withdrawal.to][withdrawal.tokenID][withdrawal.amount][withdrawal.nonce] = withdrawal.to;
```

Comment:
use a special value 1 to reduce gas cost?

Reply:
Will be the same cost, I also implemented it like this. :)